inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/InAppMessaging.kt

```diff
@@ -126,6 +127,10 @@ abstract class InAppMessaging internal constructor() {
             instance = InApp(context, isDebugLogging, isCacheHandling = isCacheHandling)
             // Initializing SDK using background worker thread.
             Initializer.initializeSdk(context, subscriptionKey, configUrl, isForTesting)
+
+            // inform event repository that it is initial launch to remove previously stored persistent event
+            LocalEventRepository.isLaunch = true
```

Comment:
Could this be called `isInitialLaunch` to make it clearer that we don't empty the cache at normal launch?

Reply:
Renamed. Please confirm.